recipes/cargo-llvm-cov/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.6.9" %}
+
+package:
+  name: cargo-llvm-cov
+  version: {{ version }}
+
+source:
+  url: https://github.com/taiki-e/cargo-llvm-cov/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 0a1b63922cfb6f2526a61bc5e13868c3f3345aa3af7337df07a69eab3d4b243e
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - {{ compiler('c') }}  # [unix]
+    - {{ stdlib('c') }}    # [unix]
```

Comment:
Why do we need a C-compiler on unix but not windows? Does the package do anything differently between unix/win?

It's worth noting that the rust compiler brings a C-compiler in any case; similarly, stdlib should be present everywhere.

Reply:
Without the compiler, I'm getting the following error:

```
Traceback (most recent call last):
  File "/home/conda/staged-recipes-copy/.ci_support/build_all.py", line 261, in <module>
    build_all(os.path.join(root_dir, "recipes"), args.arch)
  File "/home/conda/staged-recipes-copy/.ci_support/build_all.py", line 151, in build_all
    build_folders(recipes_dir, folders, arch, channel_urls)
  File "/home/conda/staged-recipes-copy/.ci_support/build_all.py", line 207, in build_folders
    conda_build.api.build([recipe], config=get_config(arch, channel_urls))
  File "/opt/conda/lib/python3.10/site-packages/conda_build/api.py", line 250, in build
    return build_tree(
  File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 3762, in build_tree
    packages_from_this = build(
  File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2839, in build
    newly_built_packages = bundlers[pkg_type](output_d, m, env, stats)
  File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 1974, in bundle_conda
    files = post_process_files(metadata, initial_files)
  File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 1782, in post_process_files
    post_build(m, new_files, build_python=python)
  File "/opt/conda/lib/python3.10/site-packages/conda_build/post.py", line 1729, in post_build
    check_overlinking(m, files, host_prefix)
  File "/opt/conda/lib/python3.10/site-packages/conda_build/post.py", line 1554, in check_overlinking
    return check_overlinking_impl(
  File "/opt/conda/lib/python3.10/site-packages/conda_build/post.py", line 1531, in check_overlinking_impl
    raise OverLinkingError(overlinking_errors)
conda_build.exceptions.OverLinkingError: overlinking check failed 
['  ERROR (cargo-llvm-cov,bin/cargo-llvm-cov): $RPATH/libgcc_s.so.1 not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?']
```

Reply:
In my opinion this is an [issue](https://github.com/conda-forge/rust-activation-feedstock/issues/51) with the rust activation that should be fixed there, once, and not worked around in many rust-recipes by adding `{{ compiler("c") }}  # [linux]`.